### ISSUE TITLE
Chat avatars lack appropriate role attributes

### ISSUE THREAD

**Issue by Woebin** [JohnXLivingston/peertube-plugin-livechat#723]
**Describe the bug**
This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 4.1.2 (A) Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value).

For SVG images to work as expected, they need to have an appropriate `role` attribute value - either `application`, `document`, or `image. The user avatars in the chat have no `role` attribute at all.

![Avatar images without role.](https://github.com/user-attachments/assets/8f4eed9c-c7ba-4727-a240-6a5f3ed8e823)

The avatars do have `aria-label` attributes, which may not be properly parsed by assistive technologies without an associated `role`.

**Expected behavior**
The avatars should have `role="image"` to go along with the `aria-label`.

**Server:**
 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: 14.0.0

**Desktop:**
 - OS: MacOS 15.5
 - Browser: Google Chrome Version 137.0.7151.120 (Official Build) (arm64)

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).
